scripts/scraper-ng/test/framework/utils.js

```diff
@@ -66,7 +66,57 @@ function recipePath(recipeName) {
   return path.join(recipesDir, recipeName + ".yaml");
 }
 
+/**
+ * Given an ingredient, an ingredient name and a tag name, check that:
```

Comment:
Perhaps this could be a little more clear about what's meant as "an ingredient" here (e.g., a `file.data.ingredient` object)?

Reply:
This is a bit tricky because `file.data.ingredient` doesn't actually exist (it's an array of `file.data.ingredients` objects). It seemed more helpful to describe what they are, but extending the description in this bit adds a lot of repetition with the `@param` line, which just looks silly. So I opted to keep this short, and extended the `@param` line a bit.

Reply:
Ah, got it. This makes sense.